README.rst

```diff
@@ -70,202 +70,1466 @@ Libraries
-     - |stable|
+     -
+     -
+     - `Client Library Issues <https://github.com/googleapis/googleapis/python-bigtable/issues>`_
+   * - `Billing <https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-billing>`_
+     - stable
+     - |PyPI-google-cloud-billing|
+     -
+     -
+     - `Client Library Issues <https://github.com/googleapis/google-cloud-python/issues>`_
+   * - `Billing Budget <https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-billing-budgets>`_
+     - stable
+     - |PyPI-google-cloud-billing-budgets|
+     -
+     -
+     - `Client Library Issues <https://github.com/googleapis/google-cloud-python/issues>`_
+   * - `Binary Authorization <https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-binary-authorization>`_
+     - stable
      - |PyPI-google-cloud-binary-authorization|
-   * - `Build <https://github.com/googleapis/python-cloudbuild>`_
-     - |stable|
+     -
+     -
+     - `Client Library Issues <https://github.com/googleapis/google-cloud-python/issues>`_
+   * - `Build <https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-build>`_
+     - stable
      - |PyPI-google-cloud-build|
-   * - `Common <https://github.com/googleapis/python-cloud-common>`_
-     - |stable|
+     -
+     -
+     - `Client Library Issues <https://github.com/googleapis/google-cloud-python/issues>`_
+   * - `Certificate Manager <https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-certificate-manager>`_
+     - stable
+     - |PyPI-google-cloud-certificate-manager|
+     -
+     -
+     - `Client Library Issues <https://github.com/googleapis/google-cloud-python/issues>`_
+   * - `Channel Services <https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-channel>`_
+     - stable
+     - |PyPI-google-cloud-channel|
+     -
+     -
+     - `Client Library Issues <https://github.com/googleapis/google-cloud-python/issues>`_
+   * - `Common <https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-common>`_
+     - stable
      - |PyPI-google-cloud-common|
-   * - `Compute Engine <https://github.com/googleapis/python-compute>`_
-     - |stable|
+     -
+     -
+     - `Client Library Issues <https://github.com/googleapis/google-cloud-python/issues>`_
+   * - `Composer <https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-orchestration-airflow>`_
+     - stable
+     - |PyPI-google-cloud-orchestration-airflow|
+     -
+     -
+     - `Client Library Issues <https://github.com/googleapis/google-cloud-python/issues>`_
+   * - `Compute Engine <https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-compute>`_
+     - stable
      - |PyPI-google-cloud-compute|
-   * - `Container Analysis <https://github.com/googleapis/python-containeranalysis>`_
-     - |stable|
+     - `API Issues <None>`_
```

Comment:
We should omit the link if it's `None`. This is happening in a few places.

Reply:
This was happening because of a missing `return` in one of the functions. Should be fixed now. Let me know if you think otherwise.